### PR TITLE
Add main.rs to run it standalone

### PR DESCRIPTION
**Pull request by 6543**
if it runs in ci, it will exec successful (exit code `0`).
if it run outside ci, exec will fail (exit code `1`).

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+use is_ci::uncached;
+
+pub fn main() {
+    let is_ci = uncached();
+    println!("{}", is_ci);
```
Comment (zkat) [inline src/main.rs]: My preference is for this to use exit codes instead of output, like `true(1)` and `false(1)`.

**Comment by 6543** [comment]
@zkat is that ok for the repo?

**Comment by zkat** [comment]
I think that's fine

**Comment by 6543** [comment]
anything missing?

**Comment by zkat** [comment]
Like I said, I would rather this use exit codes instead of stdout. I'm waiting for that change.

**Comment by 6543** [comment]
your review might be still `"pending"` ... because i didn't see any message about that ... but will ajust the pull

**Comment by 6543** [comment]
@zkat done ...

File: src/main.rs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+use is_ci::uncached;
+
+pub fn main() {
+    if let is_ci = uncached() {
+        std::process::exit(0);
+    }
+    std::process::exit(1);
```
Comment (zkat) [inline src/main.rs]:
```suggestion
    let code = if uncached() { 0 } else { 1 };
    std::process::exit(code);
```

Comment (zkat) [reply src/main.rs]: your code will always return 0 right now. Please update it.

**Review by zkat — CHANGES_REQUESTED** [review]

**Comment by 6543** [comment]
well my "more compact" did broke it but your's  works ...

**Review by zkat — APPROVED** [review]

**Pull request merged**